GoodFirstIssue-ToDoApp/Screens/ItemEdit/ItemEditViewController.swift

```diff
@@ -105,6 +94,34 @@ private extension ItemEditViewController {
     }
     
     @objc func saveButtonTapped(_ sender: UIBarButtonItem) {
+        do {
+            let realm = try Realm()
+            switch mode {
+            case .create:
+                let item: Item = Item()
+                var maxId: Int { return realm.objects(Item.self).sorted(byKeyPath: "id").last?.id ?? 0 }
+                item.id = maxId + 1
+                item.name = nameTextField.text!
+                try! realm.write {
+                    realm.add(item)
+                }
+            case .edit(id: itemId):
```

Comment:
itemIDプロパティに一致したときだけ処理したいのではなく、
.editに内容された itemID を取り出したいので、

```suggestion
            case let .edit(id: itemId):
```

で良いかなと。